src/platform/forms-system/src/js/components/Telephone.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+
+// import recordEvent from 'platform/monitoring/record-event.js';
+
+const knownNumbers = {
+  '911': ['911'],
+  '8002738255': ['crisisline', 'crisis'],
+  '8003331795': ['usdirect', 'direct', 'express'],
+  '8005351117': ['ncacenter', 'nca'],
+  '8005389552': ['dslogon', 'dmdc'],
+  '8007994889': ['vetcrisistty', 'crisistty'],
+  '8008271000': ['hotline', 'vabenefits', 'benefits', '1000'],
+  '8008778339': ['helptty'],
+  '8446982311': ['va311', '311'],
+  '8552603274': ['caregiver'],
+  '8555747286': ['hrc', 'reportline'],
+  '8663632883': ['dslogontty', 'dmdctty'],
+  '8772228387': ['helpline', 'help', '222vets'],
+  '8773270022': ['mhv', 'health', 'evet'],
+  '8774243838': ['4aidvet', '4aid', 'homeless'],
+  '8884424551': ['gibill', 'edu'],
+};
```

Comment:
I think this could be clearer if the keys/values were reversed. It might be a little more verbose and lead to some redundancy of phone numbers (if we have to have multiple aliases for each number), but we could export it as a list of constants, so that it's more readable to consumers and adds a level of enforcement. I'm picturing something like -

```js
export const CONTACTS = Object.freeze({
    911: '911'
    CRISIS_LINE: '8002738255',  
    NCA: '8005351117',
   // ....
});
```

Then the consumer would look like -

```jsx
import Telephone, { CONTACTS } from 'platform/forms-system/src/js/components/Telephone';

<Telephone contact={CONTACTS.DS_LOGON_TTY} /* other props */ >
    Contact DS Logon TTY
</Telephone>
```

If we're confident we can define all the numbers, we can add this as a prop type -

```js
Telephone.propTypes = {
    contact: PropTypes.oneOf(Object.values(CONTACTS)).isRequired,
}
```

Reply:
Oh, I like the idea of exporting the contacts! Thanks!

I don't think `<Telephone>` will have any child nodes (e.g. "Contact DS Logon TTY"), since the content will be the formatted phone number and nothing else. I didn't see any examples on the site with anything else.

I also don't think we can cover every number. I didn't include the Denver Logistics Center (DLC) numbers for ordering batteries & accessories. I'm sure there will be more numbers that need to be added in the future - I think some VA's have their own toll-free numbers, but will those subdomains even use this component?

Reply:
Ah okay, so maybe we can't enforce at the `propTypes` level. It would be neat if we could keep that list of contacts as the source of truth for VA numbers, but either way, we may need flexibility for custom/non-centralized numbers

Reply:
Implemented. I didn't include the `isRequired` prop type... maybe someday?